.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
ESLint config. Pretty barebones. But does the trick! Configuring ESLint can be kind of a pain in the ass honestly.